activesupport/lib/active_support/ordered_options.rb

```diff
@@ -14,6 +15,13 @@ module ActiveSupport
   #   h.girl = 'Mary'
   #   h.boy  # => 'John'
   #   h.girl # => 'Mary'
+  #   h.dog  # => nil
+  #
+  # To raise a exception when the value is blank append a
+  # bang to the key name, like:
+  #
+  #   h.dog! # => KeyError is raised
```

Comment:
Could you put "# => raises KeyError" to be consistent with the existing documentation please ?